src/metrics/policy_evaluations_latency.rs

```diff
@@ -0,0 +1,18 @@
+use crate::metrics::PolicyEvaluation;
+use lazy_static::lazy_static;
+use opentelemetry::{metrics::ValueRecorder, KeyValue};
+use std::time::Duration;
+
+lazy_static! {
+    static ref POLICY_EVALUATION_LATENCY: ValueRecorder<f64> =
+        opentelemetry::global::meter(super::METER_NAME)
+            .f64_value_recorder("kubewarden_policy_evaluation_latency_seconds")
+            .init();
+}
+
+pub fn record_policy_latency(latency: Duration, policy_evaluation: PolicyEvaluation) {
+    POLICY_EVALUATION_LATENCY.record(
+        latency.as_secs_f64(),
```

Comment:
Just a comment: is secs a good resolution? Shouldn't we use ms instead?

Reply:
And rename the metric ofc, I think we should aim for milliseconds overall.

Reply:
yeah, +1 for using milliseconds